### ISSUE TITLE
Does this package need interned connection

### ISSUE THREAD

**Issue by iibarant** [JoshData/python-email-validator#85]

**Comment by iibarant** [comment]
First of all, this is a great and easy to use package. 
My only concern if whether internet connection is needed while using with check_deliverability==True  ?

If the internet access is needed and dropped during processing, does that mean that the domain would not be validated?

Thank you very much!

**Comment by JoshData** [comment]
Yes, definitely, it does a DNS lookup when check_deliverability=True. If there's any sort of connection error while checking (including no internet connection), it probably will raise an EmailUndeliverableError exception.

> First of all, this is a great and easy to use package.

Thanks!
